stagecraft/apps/dashboards/tests/views/test_module.py

```diff
@@ -140,9 +127,15 @@ def test_get_module_by_uuid_404s_when_user_not_owner_of_dashboard(self):
 
         assert_that(resp.status_code, is_(equal_to(404)))
 
-    def test_modules_on_dashboard_doesnt_delete(self):
+    def test_module_doesnt_delete(self):
+        module1 = ModuleFactory(
+            type=self.module_type,
+            dashboard=self.dashboard,
+            slug='module-1',
+            options={},
+            order=1)
         delete_resp = self.client.delete(
-            '/dashboard/{}/module'.format(self.dashboard.slug),
+            '/module/{}'.format(module1.id),
```

Comment:
I think Tom and I were thinking we need to check the 405 on '/dashboard/{}/module'.format(self.dashboard.slug) rather than on the modules directly.